packages/babel-preset-expo/src/index.ts

```diff
@@ -169,6 +172,16 @@ function babelPresetExpo(api: ConfigAPI, options: BabelPresetExpoOptions = {}):
     extraPlugins.push(expoRouterBabelPlugin);
   }
 
+  if (isFastRefreshEnabled) {
+    extraPlugins.push([
+      require('react-refresh/babel'),
```

Comment:
We import ` require('react-refresh/babel')`, but we haven't added this dependency to our **package.json**. How is this being installed?

Reply:
Through react-native and metro 😭 there would be a big issue if these got out of sync.

Reply:
We might need an optional peer dep reference for pnpm in that case, I can test that if we release it as alpha (its not really testable in this monorepo)